controllers/stripe.js

```diff
@@ -0,0 +1,77 @@
+const querystring = require('querystring');
+const stripe = require('stripe')(/* CLIENT KEY */"")
```

Comment:
Perhaps it would be more practical to fetch the client key from an environment variable.

Reply:
I agree. Let's move this to an issue for later.